pxl_scripts/px/redis_data/redis_data.pxl

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Redis Data Tracer
+
+This script traces all Redis data on the cluster.
+'''
+
+import px
+
+
+def redis_data(start: str, num_head: int):
+
+    # redis_events.beta is a beta table, which is subject to change.
+    df = px.DataFrame(table='redis_events.beta', start_time=start)
+    df = add_source_dest_columns(df)
+
+    # Filter source column.
+    # df = df[df.source == 'default/cartservice']
+
+    # Restrict number of results.
+    df = df.head(num_head)
+
+    df = add_source_dest_links(df, start)
+    df = df['time_', 'source', 'destination', 'remote_port', 'cmd',
```

Comment:
this currently works (we have an issue filed) but is technically not valid pandas, can you change to double [] in case we fix it?